packages/rich-text-editor/theme/lumo/vaadin-rich-text-editor-styles.js

```diff
@@ -3,9 +3,35 @@ import '@vaadin/vaadin-lumo-styles/sizing.js';
 import '@vaadin/vaadin-lumo-styles/spacing.js';
 import '@vaadin/vaadin-lumo-styles/style.js';
 import { color } from '@vaadin/vaadin-lumo-styles/color.js';
+import { overlay } from '@vaadin/vaadin-lumo-styles/mixins/overlay.js';
 import { typography } from '@vaadin/vaadin-lumo-styles/typography.js';
 import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
+const popupOverlay = css`
+  [part='overlay'] {
+    margin: var(--lumo-space-xs) 0;
+  }
+
+  [part='content'] {
+    padding: var(--lumo-space-xs);
+    max-width: calc(7 * (1.25rem + 6px));
```

Comment:
This would be easier to read with CSS variables:
<img width="555" alt="Screenshot 2024-05-13 at 12 55 03" src="https://github.com/vaadin/web-components/assets/1222264/8f243e34-650a-42c5-ba4c-5c8a38b097dd">

Reply:
Thanks, done.